lib/addStyles.js

```diff
@@ -102,14 +102,13 @@ function addStylesToDom (styles, options) {
 		if(domStyle) {
 			domStyle.refs++;
 
-			for(var j = 0; j < domStyle.parts.length && item.parts.length; j++) {
```

Comment:
@michael-ciniawsky have we minimal test repo? maybe we can determine when this happens and add a tests